### ISSUE TITLE
Fatal error: Call to undefined method SassBoolean

### ISSUE THREAD

**Issue by yumyo** [roborourke/wp-sass#6]
After some configuration attempts all the compiling error seem solved, but now I'm facing the following 
`Fatal error: Call to undefined method SassBoolean::length() in ../wp-content/plugins/wp-sass/phpsass/script/SassScriptFunctions.php on line 874` cache and debug are set to true.

Any help will be greatly appreciated

**Comment by yumyo** [comment]
line 874 of SassScriptFunctions.php _"List Functions - taken mostly from Compass"_ ... so may the problem be the way I import compass? I'm using 'load_paths' => array("compass", "partials/susy", "partials/modular-scale") in wp-sass.php and `@import "compass";` in my .scss file

**Comment by roborourke** [comment]
I'm a little confused, have you modified wp-sass.php?

**Comment by yumyo** [comment]
yep, isn't it the way to use compass? Now switched back to the original wp-sass.php and still getting the same error.

**Comment by roborourke** [comment]
No you should need to modify wp-sass at all. Can you not just enqueue or @import the compass dependencies?

**Comment by yumyo** [comment]
Ok perfect. I've found that what is causing the problem is trying to @import susy. Looking at http://susy.oddbird.net/guides/upgrade-1-0/ seems that the minimum requirement for susy to work is Sass 3.2.0. 
Which probably it is mainly used by the at-breakpoint mixins.

**Comment by roborourke** [comment]
Aha! Thanks for finding that out. I'll see if there's an update for phpsass, until then though susy will have to wait :/

**Comment by roborourke** [comment]
@yumyo hey! I've just rolled out the updated phpsass submodule, can you try this again?

**Comment by kenorb** [comment]
Fixed in https://github.com/richthegeek/phpsass/issues/108
